azure/README.md

```diff
@@ -184,6 +184,16 @@ docker build -t azure \
 -f Dockerfile .
 ```
 
+## Build CAA Container Image
+
+Run the following set of commands from the root of this repository:
+
+```bash
+export registry="REPLACE_ME" # e.g. quay.io/username
+export CLOUD_PROVIDER=azure
+make image
+```
```

Comment:
I think you can link to the main doc for building the CAA image - https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/install/README.md#building-custom-cloud-api-adaptor-image
`make image` from the current folder will build the pod VM image

Reply:
Imo it is good practice to provide these steps in this document. It is much easier to follow a document sequentially than having to jump from one spot to another all the time.

Also, the text about the code states that it should be run from the root directory, but maybe it is a good idea to add a `cd ..`, as the following steps also assume to be executed in the root dir.

Reply:
I agree with @katexochen that follow a document sequentially is better but, in this case, I would adopt @bpradipt 's suggestion. IMHO is that in this project is better to have common documentation for common procedures/code than have it pulverized on various docs.

Reply:
This got me thinking. We can ask users to use the pre-built image. For developers we can point them to the existing documentation. WDYT folks?

The existing configs needs changes. Like this section should be commented out.
https://github.com/confidential-containers/cloud-api-adaptor/blob/a9941bba4692c22fada21b5b9f70a1986c74690d/install/overlays/azure/kustomization.yaml#L7-L10

And here we specify the image name like `quay.io/confidential-containers/cloud-api-adaptor:latest`: https://github.com/confidential-containers/cloud-api-adaptor/blob/a9941bba4692c22fada21b5b9f70a1986c74690d/install/yamls/caa-pod.yaml#L26

IIUC the images are built on each merge to `staging`. This way less steps for someone just trying stuff out for the first time.

Reply:
I guess, this may not be the best idea. As a project, we are in this stage where we want to support the developers and the users from the same documentation. So we have to do this balancing act of not leaving both out but also keeping the complexity smaller. So I made latest change where I redirect someone to the canonical documentation of image building if necessary, otherwise they use the default image.